parser_connector.py

```diff
@@ -226,7 +226,10 @@ def _handle_email(self, action_result, vault_id, label, container_id, run_automa
 
         container_id = response['container_id']
 
-        action_result.update_summary({"container_id": container_id})
+        summary = action_result.update_summary({})
+        summary['artifacts_found'] = len(response['artifacts'])
+        summary['artifacts_ingested'] = len(response['successful_artifacts'])
+        summary['container_id'] = container_id
```

Comment:
The  new fields here are consistent with other File parsing. See https://github.com/splunk-soar-connectors/parser/blob/748568d4d8965cb9c90699fa71f5cd9e5b669ca7/parser_connector.py#L429